Packs/Azure-Enrichment-Remediation/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "Azure Enrichment and Remediation",
     "description": "Playbooks using multiple Azure content packs for enrichment and remediation purposes",
     "support": "xsoar",
-    "currentVersion": "1.1.17",
+    "currentVersion": "1.1.18",
```

Comment:
should we have some other (at least optional) dependencies in here?

Reply:
Are dependencies a requirement, if so multiple others are missing.

Reply:
I think we should list what we have at least as optional

Reply:
thanks